src/server/scripts/Zandalar/Underrot/underrot.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * This file is part of the TrinityCore Project. See AUTHORS file for Copyright information
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef DEF_UNDERROT_H_
+#define DEF_UNDERROT_H_
+
+#include "CreatureAIImpl.h"
+
+#define DataHeader "Underrot"
+#define UnderrotScriptName "instance_underrot"
```

Comment:
Same thing here, URScriptName in that case.

Reply:
doesnt matter much, compiler is directly inserting the string directly anyway